geomstats/_backend/pytorch/__init__.py

```diff
@@ -51,14 +50,12 @@ def _raise_not_implemented_error(*args, **kwargs):
     raise NotImplementedError
 
 
-flip = _raise_not_implemented_error
-hsplit = _raise_not_implemented_error
 searchsorted = _raise_not_implemented_error
 vectorize = _raise_not_implemented_error
 
 
-def empty(shape, dtype=float64):
-    return torch.empty(*shape, dtype=dtype)
+def empty(tensor_shape, dtype=float64):
```

Comment:
I know this masks the module-level `shape` function otherwise, but the name was chosen to match the numpy interface. I think we need to decide in general how we are gonna handle argument names so it's possible to use kwargs consistently. A lot of the functions we simply re-export or alias have different parameter names, which prevents this from working properly.

Reply:
I did that to try and reduce the number of Deepsources flags, but I agree this feels weird.

Also, Deepsource says `torch.tensor` is not supposed to be callable, any idea why ?

Reply:
Best guess would be that it's a false alarm.

Reply:
Yes, DeepSource is not perfectly configured yet. Let me know which errors don't make sense and I can deactivate them for a whole file.

Reply:
For now I rollbacked on the change of `shape` to `tensor_shape`. Indeed, a decision must be made concerning these names. Here it's the first argument, so maybe it would be less problematic ?

Reply:
Yes, that's what I was thinking as well just now actually, see also #540.